README.md

```diff
@@ -228,6 +228,8 @@ Take a look at the following overview of the Admin App extension samples we prov
 
 * [Admin App custom theme](./src/theme#custom-theme-for-sitefinity-cms-admin-app) - You can customize the appearance of the Admin App by modifying specific components of the user interface. For example, you can customize buttons’ color, background, and text, as well as other supplementary text on the UI.
 
+* [Custom components loaded in the tree](./src/tree#custom-components-loaded-in-the-tree) - This extension is used to replace a part of each item shown in the tree used throughout the AdminApp. Currently supported only for the related data functionality.
+
```

Comment:
Users do not have a notion of tree and do not know where are we using it. Lets rename it to Custom list components

Reply:
sure, i will change it

Reply:
Suggestion: This extensibility point is used to cistomize the visualization of items in the lists throughout the AdminApp. You can provide a custom component to replace the default list node template. Currently supported only for the related data functionality.